doc/psyclone_script.rst

```diff
@@ -48,36 +53,30 @@ by the script:
     -l, --limit           limit the fortran line length to 132 characters
     -dm, --dist_mem       generate distributed memory code
     -nodm, --no_dist_mem  do not generate distributed memory code
+    --profile {invokes,kernels}, -p {invokes,kernels}
+                        Add profiling hooks for either 'kernels' or 'invokes'
+    --force-profile {invokes,kernels}
+                        Add profiling hooks for either 'kernels' or 'invokes'
+                        even if a transformation script is used. Use at your
+                        own risk.
+    -v, --version         Display version information (1.6.0)
 
 Basic Use
 ---------
 
-The simplest way to use generator.py is to provide it with an
-algorithm file.
-::
+The simplest way to use ``psyclone`` is to provide it with an
+algorithm file::
 
-    > python <PSYCLONEHOME>/src/generator.py alg.f90
+    > psyclone alg.f90
 
 If the algorithm file is invalid for some reason, the script should
 return with an appropriate error. For example, if we use the Python
-generator code itself as an algorithm file we get the following:
-::
-
-    > cd <PSYCLONEHOME>/src
-    > python ./generator.py generator.py
-    'Parse Error: Error, program, module or subroutine not found in ast'
+``genkernelstub`` script as an algorithm file we get the following::
 
-.. warning::
-
-    In the current version of PSyclone an unhelpful error ending with
-    the following may occur
-    ::
-
-        AttributeError: 'Line' object has no attribute 'tofortran'
-
-    This is due to the parser failing to parse the algorithm code and
-    is very likely to be due to the algorithm code containing a syntax
-    error.
+    > cd <PSYCLONEHOME>/bin
+    > psyclone genkernelstub
+    ...
+    'Parse Error: Fatal error in external fparser tool'
```

Comment:
When running `$ psyclone genkernelstub` from my `<PSYCLONEHOME>/bin` PSyclone installation directory I get the following error:
```
Traceback (most recent call last):
  File "./psyclone", line 39, in <module>
    from psyclone.generator import main
ImportError: No module named psyclone.generator
```

Reply:
Did you have psyclone installed? I've updated the example to make it clear that the command does not have to be run from within the bin directory (and thus you do need to have installed PSyclone first).

Reply:
Thanks for the update. I checked it again with psyclone on PATH and it returns the Parse Error as described. Running from bin directory was a bit cinfusing.